src/main/resources/application.yml

```diff
@@ -1,3 +1,22 @@
-liquibase:
-  enabled: true
-  change-log=classpath: /db/changelog/db.changelog-master.xml
+spring:
+  liquibase:
+    enabled: false
+    change-log=classpath: /db/changelog/db.changelog-master.yaml
+  datasource:
+    url: jdbc:mysql://localhost:3306/afetdb?createDatabaseIfNotExist=true
+    username: root
+    password: ippavlova_1990
```

Comment:
we need to get this info from somewhere else: ideally from vault but for now we can use uncommitted config yml file or better an environment variable defined locally and not committed

Reply:
Let's do that when we properly configure the environment.